lib/railway_ipc/ipc/consumers/republished_messages_consumer.ex

```diff
@@ -0,0 +1,30 @@
+defmodule RailwayIpc.Ipc.RepublishedMessagesConsumer do
+  use RailwayIpc.CommandsConsumer,
+    queue: "railway_ipc:republished_messages:commands",
+    publish_function: &RailwayIpc.Publisher.publish/1
```

Comment:
This is the publish function specification described in my earlier comment. Do we want to avoid this?